openeo/Pre&PostCovid.ipynb

```diff
@@ -0,0 +1,358 @@
+      "0:03:02 Job 'j-2401094851a04db380f7a9813a2b4e61': running (progress N/A)\n",
+      "0:03:39 Job 'j-2401094851a04db380f7a9813a2b4e61': running (progress N/A)\n",
+      "0:04:26 Job 'j-2401094851a04db380f7a9813a2b4e61': running (progress N/A)\n",
+      "0:05:24 Job 'j-2401094851a04db380f7a9813a2b4e61': running (progress N/A)\n",
+      "0:06:25 Job 'j-2401094851a04db380f7a9813a2b4e61': running (progress N/A)\n",
+      "0:07:25 Job 'j-2401094851a04db380f7a9813a2b4e61': finished (progress N/A)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Lets check if it is good or not\n",
+    "job = s5covid.execute_batch(title=\"NO2 during Covid\", outputfile=\"during_covid.nc\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "1e25702f-6388-461d-a865-0af22f37bc52",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0:00:00 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': send 'start'\n",
+      "0:00:23 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': created (progress N/A)\n",
+      "0:00:29 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': created (progress N/A)\n",
+      "0:00:35 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': created (progress N/A)\n",
+      "0:00:43 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': created (progress N/A)\n",
+      "0:00:53 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': running (progress N/A)\n",
+      "0:01:38 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': running (progress N/A)\n",
+      "0:01:53 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': running (progress N/A)\n",
+      "0:02:13 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': running (progress N/A)\n",
+      "0:02:37 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': running (progress N/A)\n",
+      "0:03:07 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': running (progress N/A)\n",
+      "0:03:44 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': running (progress N/A)\n",
+      "0:04:31 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': running (progress N/A)\n",
+      "0:05:49 Job 'j-2401095f84234e3a9a9f20e23a29b5e2': finished (progress N/A)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Lets check if it is good or not\n",
+    "job = s5post.execute_batch(title=\"NO2 Post-Covid\", outputfile=\"post_covid.nc\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ceef1400-e882-4fc5-881d-4a0767344666",
+   "metadata": {},
+   "source": [
+    "### Let's plot the result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "e4f41bef-5136-4e51-a1b3-34782c728df8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import xarray as xr\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "b4c667b8-a4f3-4dec-b28e-56b0e911a16e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load the results\n",
+    "predata = xr.load_dataset(\"during_covid.nc\")\n",
```

Comment:
Note that you risk confusing yourself: you load *during* data as *predata*

Reply:
maintained variable as duringdata